src/components/TripPlanner/TripPlanner.tsx

```diff
@@ -1,23 +1,49 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { MapChart } from "./Map";
 import { SearchByLocation } from "./SearchByLocation";
 import { SightingsTable } from "./SightingsTable";
-import { TripPlannerRequest, getSightingsListByLocation, TripPlannerResponse } from "../../clients/apiClient";
+import { TripPlannerRequest, getTopFiveSightingsByLocation, TripPlannerResponse, WhaleSighting } from "../../clients/apiClient";
 import './TripPlanner.scss'
 
 export function TripPlanner() {
 
     const [latLon, setLatLon] = useState<TripPlannerRequest>({ latitude: 0, longitude: 0 });
+    const [listSightings, setListSightings] = useState<TripPlannerResponse[]>([]);
+    const [nearestFiveSightings, setNearestFiveSightings] = useState<WhaleSighting[]>([]);
+    var listNearestFiveSightings: WhaleSighting[] = [];
 
-    return <section className="tripplanner-container">
-        <div className="tripplanner-search">
-            <SearchByLocation latLon={latLon} setLatLon={setLatLon} />
-        </div>
-        <div className="tripplanner-map">
-            <MapChart />
-        </div>
-        <div className="tripplanner-table">
-            <SightingsTable latLon={latLon} />
-        </div>
-    </section>
+    useEffect(() => {
+        if (latLon) {
+            getTopFiveSightingsByLocation(latLon)
+                .then(response => { if (response) setListSightings(response); ConvertResponseToWhaleSightings(listSightings); })
+        }
+    }, [latLon]);
+
+    function ConvertResponseToWhaleSightings(listSightings: TripPlannerResponse[]) {
```

Comment:
Function names should be in lower camel case, React components are upper camel case
```suggestion
    function convertResponseToWhaleSightings(listSightings: TripPlannerResponse[]) {
```

Reply:
changed